Makefile

```diff
@@ -115,10 +115,18 @@ test: manifests generate fmt vet envtest ## Run tests.
 build: generate fmt vet ## Build manager binary.
 	go build -o bin/manager main.go
 
+.PHONY: build-4-debug
+build-4-debug: generate fmt vet ## Build manager binary with debug options.
+	go build -gcflags="all=-N -l" -o bin/manager main.go
+
 .PHONY: run
 run: manifests generate fmt vet ## Run a controller from your host.
 	go run ./main.go
 
+.PHONY: debug
+debug: build-4-debug ## Run a controller from your host from binary
+	./bin/manager
```

Comment:
what this debug here is doing?

Reply:
One run the build with the flags for debug this run the debug after the build